CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [0.0.7] - 2024-11-08
+
+### Added
+
+- Added option to the `WaitGroup` runnable to block the `Close` function on the `Waitgroup` completing
+
+## [0.0.6] - 2024-10-18
```

Comment:
dont these need a reference with a link? usually at the bottom of the file

something like [this](https://raw.githubusercontent.com/joeyagreco/sleeper/refs/heads/main/CHANGELOG.md)

Reply:
Missed that, thanks for letting me know. Updating...

Reply:
if we prefer to stick with what we have, then we should make this match the agent repo style then right?

## [0.0.7] - 2024-11-08

vs 

## v1.16.0

Reply:
I guess I wasn't trying to change the existing format of this changelog, just update it with the latest changes while maintaining its current form.

I think I'd prefer to change the format in its own change if that's ok with you, but you do make a good point that we're not consistent. We can talk with @harbolkn about this. I don't believe it's a priority, but I think he was (semi-)owning our changelog/changelog process once we commit to an approach.

Reply:
makes sense